### ISSUE TITLE
invalid ui elements

### ISSUE THREAD

**Issue by BuckarooBanzay** [LucxMangaJet/minetest_presentations#11]
When i'm trying to use the remote i get "greeted" with those errors:

```
2021-02-18 14:20:03: ERROR[Main]: Invalid label element(3): '1,0.5; ID: 1;'
2021-02-18 14:20:03: ERROR[Main]: Invalid button element(5): '5,0.25; 1.5,.5;Destroy;Destroy;'
2021-02-18 14:20:03: ERROR[Main]: Invalid button element(5): '6.5,0.25; 2.5,.5;DestroyAndCleanup;Destroy And Cleanup;'
2021-02-18 14:20:03: ERROR[Main]: Invalid label element(3): '1,1.25; Move;'
2021-02-18 14:20:03: ERROR[Main]: Invalid button element(5): '1,1.5; 1,0.5;MoveRight;X+;'
2021-02-18 14:20:03: ERROR[Main]: Invalid button element(5): '2,1.5; 1,0.5;MoveUp;Y+;'
```

**Comment by LucxMangaJet** [comment]
Hey, what minetest version are you running and on what device?

**Comment by BuckarooBanzay** [comment]
> Hey, what minetest version are you running and on what device?
> I'm trying to replicate it.

Minetest `5.4.0-dev` on commit https://github.com/minetest/minetest/commit/857dbcd5728e2f18cdbb478d85f5861d5f0c7123
Device: PC with Linux (it shouldn't matter)

**Comment by pirey0** [comment]
Alright, looks like excessive ";" in formspecs became cause for error in 5.4.0.
Fixed it in 8eafa08115a16c2d3fcbabd2d878904f347e79da
I'll update the release.
